CounterExample/ViewController.swift

```diff
@@ -41,13 +47,14 @@ extension ViewController : UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "CounterCell", for: indexPath) as! CounterCell
         cell.label.text = "\(mainStore.state.counters[indexPath.row])"
-        cell.action = { didIncrease in
-            if didIncrease {
+        cell.action = { actionType in
```

Comment:
What about putting the logic of the following switch - case into `CellActionType` itself. So later we could avoid code duplication to select the action to dispatch based on the action type. Because this one will be stored only in one place in `CellActionType` enum.

Thoughts?